packages/loader/container-loader/src/container.ts

```diff
@@ -967,6 +970,14 @@ export class Container
 			forceEnableSummarizeProtocolTree,
 		);
 
+		this.containerStateManager = new containerStateManager(
```

Comment:
i would expect this to taking pendingLocalState and whereever it is need it is retrieved from here, and here only